README.md

```diff
@@ -0,0 +1,93 @@
+# Ionic Tabs Template - Upgraded
+
+This project provides an extension to the tabs template that is provided by Ionic. It provides the following additional functionalities.
+
+- Basic desktop compatible UI
+- Cypress testing
+- WCAG 2.2 AA compliance
```

Comment:
Is this part true? I.e. full WCAG 2.2 AA compliance, i.e. that it's been assessed for manual-only testable issues too?

According to a few sources, WCAG 2.2 AA compliance can only partially tested with fully-automatic tools (~20%) and a significant proportion more with semi-automated tools (~60%), the remainder is manual testing (~20%):
- https://karlgroves.com/web-accessibility-testing-what-can-be-tested-and-how/ (2012)
- https://webaim.org/projects/million/ (2024)
- https://www.deque.com/automated-accessibility-testing-coverage/ (2024)